presto-main/src/test/java/com/facebook/presto/operator/TestFilterAndProjectOperator.java

```diff
@@ -100,7 +102,9 @@ public void test()
                 0,
                 new PlanNodeId("test"),
                 processor,
-                ImmutableList.of(VARCHAR, BIGINT));
+                ImmutableList.of(VARCHAR, BIGINT),
+                new DataSize(0, BYTE),
+                0);
```

Comment:
Consider adding a smoke tests of operator for case when pages are merged. To verify proper integration of operator code with `MergingPageOutput`.